### ISSUE TITLE
Invite Friends

### ISSUE THREAD

**Issue by himanshusharma89** [himanshusharma89/relic_bazaar#56]
Use dynamic links to share with your friends to invite. Follow up pf #50

**Comment by nb9960** [comment]
Since this is the follow up of #50 Do I have to work on this as well?

**Comment by himanshusharma89** [comment]
Yes @nb9960, but one person can be assigned to one issue at a time.

**Comment by nb9960** [comment]
> Yes @nb9960, but one person can be assigned to one issue at a time.

Okay, but featuring dynamic links using firebase requires setting up firebase first which in itself is a separate issue. Or do one thing generate URL using firebase and provide me that link so that I can start working on #50

**Comment by himanshusharma89** [comment]
@nb9960 I'll add you to the firebase console and integration is almost done, just need to review the PR. Share me your Gmail id.

**Comment by nb9960** [comment]
> @nb9960 I'll add you to the firebase console and integration is almost done, just need to review the PR. Share me your Gmail id.

nbodani8@gmail.com

**Comment by anotherwebguy** [comment]
Hey @himanshusharma89 i can work on this issue if issue #50 is merged, also can u fill me with details like here we sharing the links to the products right?
So i need to pass the products info in the link ?
Also i need to work on that link generation plus sharing part?
Also after opening app from link it should take you to the product detail page right?
Plz assign me this issue i would like to work on it.
Here's my mail u can add me on firebase console
mohitrajputms123@gmail.com

**Comment by anotherwebguy** [comment]
Hey @himanshusharma89 here do we only have to share the dynamic links that are generated in issue #50 ?

**Comment by nb9960** [comment]
> Hey @himanshusharma89 i can work on this issue if issue #50 is merged, also can u fill me with details like here we sharing the links to the products right?
> So i need to pass the products info in the link ?
> Also i need to work on that link generation plus sharing part?
> Also after opening app from link it should take you to the product detail page right?
> Plz assign me this issue i would like to work on it.
> Here's my mail u can add me on firebase console
> [mohitrajputms123@gmail.com](mailto:mohitrajputms123@gmail.com)

Hey @anotherwebguy PR for dynamic link integration is not merged yet.
For resolving this issue you just have to create a share button and pass URL as relicbazaar.page.link/ (without any parameter as this isn't about sharing product, I have already done that part). Other than that as I have already integrated dynamic link you don't have to make any changes through the firebase console or in the codebase to handle the link on app launch

**Comment by anotherwebguy** [comment]
> > Hey @himanshusharma89 i can work on this issue if issue #50 is merged, also can u fill me with details like here we sharing the links to the products right?
> > So i need to pass the products info in the link ?
> > Also i need to work on that link generation plus sharing part?
> > Also after opening app from link it should take you to the product detail page right?
> > Plz assign me this issue i would like to work on it.
> > Here's my mail u can add me on firebase console
> > [mohitrajputms123@gmail.com](mailto:mohitrajputms123@gmail.com)
> 
> Hey @anotherwebguy PR for dynamic link integration is not merged yet.
> For resolving this issue you just have to create a invite button (maybe in drawer) and pass URL as relicbazaar.page.link/ (without any parameter as this isn't about sharing product, I have already done that part). Other than that as I have already integrated dynamic link you don't have to make any changes through the firebase console or in the codebase to handle the link on app launch

Yeah @nb9960 i saw your pr i got my doubt cleared so i can work on this part right @himanshusharma89 ?

**Comment by himanshusharma89** [comment]
@anotherwebguy As I mentioned earlier one person can work on a single issue at a time.

**Comment by anotherwebguy** [comment]
> @anotherwebguy As I mentioned earlier one person can work on a single issue at a time.

Hey @himanshusharma89 but at the moment i am not assigned any issue plz assign me this issue

**Comment by himanshusharma89** [comment]
@anotherwebguy The issue is still not resolved but I'll assign new issues to you. By responsive it means we don't have to add a custom height for the widget instead use the Flex.

**Comment by anotherwebguy** [comment]
> @anotherwebguy The issue is still not resolved but I'll assign new issues to you. By responsive it means we don't have to add a custom height for the widget instead use the Flex.

Okay sir i'll do that

**Comment by anotherwebguy** [comment]
> @anotherwebguy The issue is still not resolved but I'll assign new issues to you. By responsive it means we don't have to add a custom height for the widget instead use the Flex.

So now can u assign me this issue as i've now made an pr for issue #39

**Comment by himanshusharma89** [comment]
> So now can u assign me this issue as i've now made an pr for issue #39

This issue will be taken care of later on as it is dependant upon the Dynamic Links. And we need to create the UI first.

**Comment by subhangi2731** [comment]
@himanshusharma89 please assign this issue to me
